dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -355,27 +349,14 @@ private void updateTagsWithBuildInformation(String repository, List<Tag> tags, T
     }
 
     /**
-     * Get the map of the given Quay tool
-     * Todo: this should be implemented with the Quay API, but they currently don't have a return model for this call
+     * Get the given Quay tool
      *
      * @param tool
      * @return
      */
-    public Map<String, Object> getQuayInfo(final Tool tool) {
-        final String repo = tool.getNamespace() + '/' + tool.getName();
-        final String repoUrl = QUAY_URL + "repository/" + repo;
-        final Optional<String> asStringBuilds = ResourceUtilities.asString(repoUrl, quayToken.getContent(), client);
-
-        if (asStringBuilds.isPresent()) {
-            final String json = asStringBuilds.get();
-
-            Gson gson = new Gson();
-            Map<String, Object> map = new HashMap<>();
-            map = (Map<String, Object>)gson.fromJson(json, map.getClass());
-            return map;
-
-        }
-        return null;
+    public QuayRepo getQuayInfo(final Tool tool) {
```

Comment:
Either way, but as a recent convert to this style, I would return an Optional.

Reply:
Yeah, was being lazy but you're probably right